### DISCUSSION TITLE
How can I style all connections?

### DISCUSSION THREAD

**Discussion by i-am-the-slime** [terrastruct/d2#2016] [Q&A] [answered]
I am having trouble fixing a style for all connections. `* -> *` and assigning a class leads to lots of new connections, however, I just want to change the stroke and text colour to be black. 

Is there a trick to do this?

**Comment by terlar** [comment, marked as answer]
See this about how to do it:

https://github.com/terrastruct/d2/issues/1989

TLDR `(** -> **)[*]`
